net-mgmt/snmp/pkg-descr

```diff
@@ -0,0 +1,4 @@
+The snmp plugin allows you to query information about your OPNsense appliance about
+memory, load and CPU utilization and system statistics via the bsnmp-ucd module.
+It also allows to create counters from log files, program output and other text data via the bsnmp-regex module.
```

Comment:
this is more or less what the two dependencies have as an information a bit rewritten to fit better together. Still does not sound well, but it is better than nothing...